weld/resources/merger/init_merger.ll

```diff
@@ -0,0 +1,19 @@
+  ; Copy the initial value into the first merger
+  {nworkers} = call i32 @get_nworkers()
+  {first} = call {bld_ty_str} {bld_prefix}.getPtrIndexed({bld_ty_str} {bld_inp}, i32 0)
+  store {elem_type} {init_elem}, {bld_ty_str} {first}
+  br label %{entry}
+
+{entry}:
+  {cond} = icmp ult i32 1, {nworkers}
+  br i1 {cond}, label %{body}, label %{done}
+
+{body}:
+  {i} = phi i32 [ 1, %{entry} ], [ {i2}, %{body} ]
+  {cur_ptr} = call {bld_ty_str} {bld_prefix}.getPtrIndexed({bld_ty_str} {bld_inp}, i32 {i})
+  store {elem_type} {iden_elem}, {bld_ty_str} {cur_ptr}
```

Comment:
the first thread gets the initial value (see line 4 of this file). Remaining threads get an op-specific identity value.